5.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <footer class="main-footer">
+        <div class="main-footer__header">
+            <h2 class="text--size-l">
+                Support
+            </h2>
+            <p class="text--size-s">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+            </p>
+        </div>
+        <article class="main-footer__item">
+            <p class="text--size-x">Sales</p>
+            <p class="text--size-xs">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+            </p>
+            <a href="#" class="ref">Contact us</a>
+        </article>
+        <article class="main-footer__item">
+            <p class="text--size-x">Sales</p>
+            <p class="text--size-xs">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+            </p>
+            <a href="#" class="ref">Contact us</a>
+        </article>
+        <article class="main-footer__item">
+            <p class="text--size-x">Sales</p>
+            <p class="text--size-xs">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+            </p>
+            <a href="#" class="ref"  class="text--size-x">Contact us</a>
+        </article>
```

Comment:
```suggestion
        <article class="main-footer__item service">
            <h2 class="service__title">Sales</h2>
            <p class="service__text">
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
            </p>
            <a href="#"  class="service__footer">Contact us</a>
        </article>
```
 вот так примерно сделать, и у вас много где на элементах 2 атрибута class